guide/getting-started/open-design.md

```diff
@@ -0,0 +1,112 @@
+- Bosch working with Bitcoin Core GUI (project-specific)
+
+Ideas
+- Add quotes and stories from designers
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/getting-started/open-design/open-design.jpg"
+   retina = "/assets/images/guide/getting-started/open-design/open-design@2x.jpg"
+   mobile = "/assets/images/guide/getting-started/open-design/open-design-mobile.jpg"
+   mobileRetina = "/assets/images/guide/getting-started/open-design/open-design-mobile@2x.jpg"
+   alt-text = ""
+   caption = ""
+   width = 1600
+   height = 740
+   layout = "full-width"
+%}
+
+# Open design
+
+**In a broader view, open design is an alternative method for design and development, based on the free exchange of comprehensive design information.**
+
+Open design tries to balance the independence of individual designers and the collective power of collaboration. The goal is to enable and promote design projects, which are motivated by personal conviction and passion of designers for the greater benefit of a global society.
+
+In practical terms, open design can manifest itself in a different ways. You might:
+
+* help design the interface of your favorite open-source project
+* be really interested in a specific topic and collaborate with multiple people and projects in related areas
+* create a design resource library (like an icon or illustration set, or a UI Kit) for others to use
+* help test and provide feedback to an open-source product
+* contribute to this Bitcoin Design Guide (hint, hint)
+
+Whichever it may be, it is a voluntary, public contribution to efforts that you think are worthwhile. From these individual efforts, a network of people and projects emerges that is only loosely bound through personal relationships, communities, or the work itself. This is different from common corporate structures and requires individuals to think and act more independently and be comfortable with ambiguity.
+
+Open-source development has existed and matured for decades, while open design is not as widely known or practiced. Starting to design for bitcoin products can therefore sometimes be disorienting, as many designers need to adjust not just how they work and collaborate, but also the relationship to their work. No longer is the design an artefact only controlled by just the designer or a small group of people inside an organzation. The design instead becomes part of a public conversation where anyone is invited to help shape it, and group consensus influences the way forward. In open-source software, this public review and debate often leads to better products and services, and this effect ideally carries over to open design.
+
+The best solutions typically come from a comprehensive understanding of the problem to be solved, whether they are technical, human, societal, economic, or otherwise. This makes a multidisciplinary approach essential, with users, designers, developers, translaters, project managers, and others working together.
```

Comment:
```suggestion
The best solutions typically come from a comprehensive understanding of the problem to be solved, whether they are technical, human, societal, economic, or otherwise. This makes a multidisciplinary approach essential, with users, designers, developers, translators, project managers, and others working together.
```